sub-packages/bionemo-evo2/src/bionemo/evo2/run/train.py

```diff
@@ -575,30 +835,20 @@ def train(args: argparse.Namespace) -> nl.Trainer:
             }
         else:
             activation_checkpointing_args = {}
+
     # Retrieve model config.
     config_modifiers_init = {
-        "calculate_per_token_loss": not args.no_calculate_per_token_loss,  # override megatron internal behavior.
```

Comment:
let's probably add this back, but I'm curious if this change hurts your training loss curve...